crates/winscard/src/scard.rs

```diff
@@ -342,7 +347,7 @@ impl SmartCard<'_> {
         // NIST.SP.800-73-4, Part 1, Table 5
         const RSA_ALGORITHM: u8 = 0x07;
         // NIST.SP.800-73-4, Part 1, Table 4b
-        const PIV_AUTHENTICATION_KEY: u8 = 0x9A;
+        const PIV_AUTHENTICATION_KEY: u8 = 0x9C;
```

Comment:
According to the specification, `9c` corresponds to the _Digital Signature Key_ and `9a` to the `PIV Authentication Key`.
I decided to change this value to `9c`

Reply:
I think the name of the item should be changed too then? It’s still named `PIV_AUTHENTICATION_KEY`.

Reply:
Oh, right. I forgot about it.
Fixed